src/nvim/shada.c

```diff
@@ -287,6 +288,7 @@ typedef struct {
       char **contents;
       size_t contents_size;
       size_t width;
+      bool is_unnamed;
```

Comment:
AFAIK this better be placed after `char`. Here it increases size by `sizeof(void *)` (next pointer must be aligned), but putting it after `char` while both `char` and `bool` do not require to be aligned should not even alter the total size of the `struct reg`.